tests/e2e-openshift/monolithic-multitenancy-rbac/02-install-otelcol.yaml

```diff
@@ -0,0 +1,132 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: otel-collector-deployment
+  namespace: chainsaw-mmo-rbac
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: chainsaw-mono-rbac-clusterrole
+rules:
+- apiGroups: [""]
+  resources: ["pods", "namespaces", "nodes"]
+  verbs: ["get", "watch", "list"]
+- apiGroups: ["apps"]
+  resources: ["replicasets"]
+  verbs: ["get", "list", "watch"]
+- apiGroups: ["extensions"]
+  resources: ["replicasets"]
+  verbs: ["get", "list", "watch"]
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: chainsaw-mono-rbac-clusterrole-binding
+subjects:
+- kind: ServiceAccount
+  name: otel-collector-deployment
+  namespace: chainsaw-mmo-rbac
+roleRef:
+  kind: ClusterRole
+  name: chainsaw-mono-rbac-clusterrole
+  apiGroup: rbac.authorization.k8s.io
+
+---
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: dev
+spec:
+  serviceAccount: otel-collector-deployment
+  config: |
+    extensions:
+      bearertokenauth:
+        filename: /var/run/secrets/kubernetes.io/serviceaccount/token
+
+    receivers:
+      otlp/grpc:
+        protocols:
+          grpc:
+      otlp/http:
+        protocols:
+          http:
+  
+    processors:
+      k8sattributes:
+        extract:
```

Comment:
do we need this extra config?

Reply:
Its not needed for the feature but we are using it for testing to check the user which doesn't have permission on the namespace cannot view attributes other than k8s.namespace.name